pom.xml

```diff
@@ -936,6 +944,14 @@ limitations under the License.
     </profile>
   </profiles>
   <repositories>
+    <repository>
+      <id>spring-milestones</id>
+      <name>Spring Milestones</name>
+      <url>https://repo.spring.io/milestone</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
```

Comment:
Same than https://github.com/Activiti/activiti-cloud/pull/1224#discussion_r1418776842: I wonder if it should be moved after central (?) cc @matteo4diani

Reply:
We don't need to use Spring Milestones repository after official SB release has been published  in Maven Central, so I will remove it.

Reply:
Ah! Makes more sense :+1: